src/peripheral/dwt.rs

```diff
@@ -62,7 +62,56 @@ pub struct Comparator {
     reserved: u32,
 }
 
+// DWT CTRL register fields
```

Comment:
It might be worth trying to add a link to the Arm Armv7-EM ARM here, to explain where the constants came from.

https://developer.arm.com/documentation/ddi0403/d/Debug-Architecture/ARMv7-M-Debug/The-Data-Watchpoint-and-Trace-unit/Control-register--DWT-CTRL might work.